holidayextras/ui-brand-heha
# HEHA-251

#### What does this PR do? (please provide any background)

Implements the new HEHA! theme from Nodivide as a bootstrap theme comparable with the pattern library.

#### What tests does this PR have?

N/A

#### How can this be tested?

This PR needs to be reviewed in conjunction with:
https://github.com/holidayextras/pattern-library/commit/1d67e8ba16558ca8f03b480d32216c57751169ee

Check both of these branches out next to each other and from the pattern library directory run:

```
npm install // If you haven't already
npm link ../ui-brand-heha
npm start
```

Pull up localhost:3000 and have a butchers. The main elements that we care about for HEHA are the planner page and list groups.

This theme has responsive styles to test.

#### Screenshots / Screencast

![](https://cldup.com/J5BNX7JMf9.png)
![](https://cldup.com/yDyKs1_1Mn.png)
![](https://cldup.com/Jj25NxB6Zx.png)
![](https://cldup.com/WHiYLF8mto.png)

#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/PkX6O4vfRwjao/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
